.buildkite/test.sh

```diff
@@ -60,7 +60,7 @@ docker run --rm alpine sh -c '
   done
 '
 
-rm -rf ~/.ddev/Test* ~/.ddev/global_config.yaml ~/.ddev/homeadditions ~/.ddev/commands
+rm -rf ~/.ddev/Test* ~/.ddev/global_config.yaml ~/.ddev/homeadditions ~/.ddev/commands ~/.ddev/bin/docker-comnpose*
```

Comment:
small typo here *comnpose*

Reply:
I guess that's not going to work out! Thanks for paying attention! :)

Reply:
```suggestion
rm -rf ~/.ddev/Test* ~/.ddev/global_config.yaml ~/.ddev/homeadditions ~/.ddev/commands ~/.ddev/bin/docker-compose*
```

Reply:
This typo somehow made its way to master branch :)